src/content/en/updates/2020/11/cvd.md

```diff
@@ -0,0 +1,461 @@
+
+- The Inspect Mode tooltip that appears on top of the web page [shows the contrast ratio](/web/updates/2020/05/devtools#a11y) for text elements.
+- The DevTools color picker [calls out bad contrast ratios](/web/updates/2018/01/devtools#contrast) for text elements, [shows the recommended contrast line](/web/updates/2019/01/devtools#AAA) to help manually select better colors, and can even [suggest accessible colors](/web/updates/2020/08/devtools#accessible-color).
+- Both [the CSS Overview panel](/web/updates/2020/10/devtools#css-overview) and [the Lighthouse Accessibility audit report](https://web.dev/color-contrast/) lists low-contrast text elements as found on your page.
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="Mje2wYgPYP0"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+We’ve recently added a new tool to this list, and it’s a bit different from the others. The above tools mainly focus on *surfacing contrast ratio information* and giving you options to *fix* it. We realized that DevTools was still missing a way for developers to get a deeper *understanding* of this problem space. To address this, we implemented [vision deficiency simulation](/web/updates/2020/03/devtools#vision-deficiencies) in the DevTools Rendering tab.
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="mK_XmFb8E_w"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+In Puppeteer, [the new `page.emulateVisionDeficiency(type)` API](https://github.com/puppeteer/puppeteer/blob/v5.4.1/docs/api.md#pageemulatevisiondeficiencytype) lets you programmatically enable these simulations.
+
+## Color vision deficiencies {: #cvd }
+
+[Roughly 1 in 20 people](http://www.colourblindawareness.org/colour-blindness/) suffer from a color vision deficiency (also known as the less accurate term “color blindness”). Such impairments make it harder to tell different colors apart, which *can amplify contrast issues*.
+
+
+<figure>
+    <picture>
+      <!-- <source srcset="/web/updates/images/2020/11/cvd/0-regular@2x.avif 2x, /web/updates/images/2020/11/cvd/0-regular.avif 1x" type="image/avif" />
+      <source srcset="/web/updates/images/2020/11/cvd/0-regular@2x.webp 2x, /web/updates/images/2020/11/cvd/0-regular.webp 1x" type="image/webp" /> -->
+      <img srcset="/web/updates/images/2020/11/cvd/0-regular@2x.png 2x, /web/updates/images/2020/11/cvd/0-regular.png 1x"
+        alt="A colorful picture of melted crayons, with no color vision deficiencies simulated"/>
+    </picture>
+  <figcaption>
+    <b>Figure 1</b>. Description: A colorful <a href="https://unsplash.com/photos/keMdIzsNzsM">picture of melted crayons</a>, with no color vision deficiencies simulated.
```

Comment:
Sorry for the confusion — I left these `Figure X: Description:` notes in the doc just to clarify how I planned to use it. Since we're using `<figcaption>` it's already clear these are descriptions so let's drop this:

```suggestion
    A colorful <a href="https://unsplash.com/photos/keMdIzsNzsM">picture of melted crayons</a>, with no color vision deficiencies simulated.
```